GitHub_Learning/Feature/Login/View/LoginViewController.swift

```diff
@@ -31,15 +31,30 @@ class LoginViewController: BaseViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        loginLayout.loginButton.addTarget(self, action: #selector(loginButtonPressed), for: .touchUpInside)
+        layout.passwordTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        layout.emailTextField.addTarget(self, action: #selector(textFieldDidChange(_:)), for: .editingChanged)
+        layout.loginButton.addTarget(self, action: #selector(loginButtonPressed), for: .touchUpInside)
         loginPresenter.viewDidLoad()
     }
     
     @objc private func loginButtonPressed() {
-        loginPresenter.logInWith(email: loginLayout.emailTextField.text, password: loginLayout.passwordTextField.text)
+        loginPresenter.logIn()
+    }
+    
+    @objc private func textFieldDidChange(_ textField: UITextField) {
+        if textField.tag == 0 {
```

Comment:
Tady se muzes rovnou ptat takle:
```suggestion
        if textField.tag == layout.emailTextField {
```
A nepotrebujes nastavovat ty tagy.
Dokonce muzes pouzit identitu `textField.tag === layout.emailTextField`.

Reply:
`textField.tag === layout.emailTextField` ani `textField.tag == layout.emailTextField`, dal jsem `textField.tag == layout.emailTextField.tag`.

Reply:
Sorry, napsal jsem ti to tam blbe. Ma to byt takle 😄 `textField == layout.emailTextField`.